lib/gobstones/batch_parser.rb

```diff
@@ -32,7 +32,7 @@ def parse_example(options, example)
 
       return_value = example[:postconditions][:return]
       if return_value
-        example[:title] = "#{options[:subject]}() -> #{return_value}" unless example[:title]
+        example[:name] = "#{options[:subject]}() -> #{return_value}" unless example[:name]
```

Comment:
@rodri042 was it `name` actually?

Reply:
Yes, Metatest expects `name`, but in the examples YAML it's `title`